src/core/layout/scrollable-list/button-overlays.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @module core/layout/scrollable-list
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0 Apache-2.0
+ */
+import fp from "../../../../lib/lodash/fp/fp.js";
+
+export const overlays1Wide = ({ scene, gelButton, item, config }) => {
```

Comment:
Super minor but we usually put exports at the bottom.
Don't think this is written down anywhere.  Might install an eslint plugin for this.

Reply:
33026bf